src/memray/_memray.pyi

```diff
@@ -133,7 +133,10 @@ class FileReader:
     @property
     def metadata(self) -> Metadata: ...
     def __init__(
-        self, file_name: Union[str, Path], *, report_progress: bool = False
+        self,
+        file_name: Union[str, Path],
+        *,
+        report_progress: bool = False,
```

Comment:
Didn't you mean to add `max_memory_records` here?

Reply:
Yup, something went wrong with the rebase :(